lib/ansible/executor/task_executor.py

```diff
@@ -728,6 +728,11 @@ def _evaluate_failed_when_result(result):
             for k in plugin_vars:
                 result["_ansible_delegated_vars"][k] = cvars.get(k)
 
+            # note: here for callbacks that rely on this info to display delegation
+            for requireshed in ('ansible_host', 'ansible_port', 'ansible_user', 'ansible_connection'):
```

Comment:
`requireshed`?

Reply:
current 'opinion' of what is required, probably open to bikeshed